packages/components/src/VirtualizedList/RendererSelector.component.js

```diff
@@ -24,6 +25,13 @@ function getRowRenderer(type) {
 	return rowRenderer;
 }
 
+export function getNoRowsRenderer(t, NoRowsRenderer) {
+	if (NoRowsRenderer) {
+		return () => <NoRowsRenderer />;
+	}
+	return () => <NoRows t={t} />;
```

Comment:
Instead of creating this function, I think it would be better to 
* wrap the default NoRows with a translate instead of passing `t` like the old code
* use default props to set NoRows (without `t` to pass :point_up:)